src/System.Text.JsonLab/System/Text/Json/JsonReader.cs

```diff
@@ -622,15 +664,15 @@ private void ConsumeSingleValue(ref ReadOnlySpan<byte> buffer, byte marker)
             }
         }
 
-        private void ConsumeNumberUtf8MultiSegment()
+        private void ConsumeNumberUtf8MultiSegment(ref BufferReader<byte> reader)
         {
-            if (!_reader.TryReadToAny(out ReadOnlySpan<byte> span, JsonConstants.Delimiters, advancePastDelimiter: false))
+            //TODO: Increment Index by number of bytes advanced
+            if (!reader.TryReadToAny(out ReadOnlySpan<byte> span, JsonConstants.Delimiters, advancePastDelimiter: false))
```

Comment:
@JeremyKuhne, I need TryReadToAny to return how many elements the BufferReader was advanced by to increment the Index property within the JsonReader. Is that possible?
https://github.com/dotnet/corefxlab/blob/8cd063a1e163a0db906343bb5ba1b53c02eb83d6/src/System.Buffers.ReaderWriter/System/Buffers/Reader/BufferReader_search.cs#L100

Reply:
Does Consumed provide what you need?

Reply:
Not completely since consumed is the total data consumed rather than just the elements that were advanced by the particular operation. I could compare the `Consumed` property before/after to see how much it changed, as a workaround. Is that what you would expect others to do?

Reply:
> Is that what you would expect others to do?

Only if there were perf questions involved that made returning the diff a significant tax. I'm otherwise unopposed to always returning count, and would prefer it be consistent.